LICENSE.md

```diff
@@ -0,0 +1 @@
+debian/copyright
```

Comment:
same as above, we could move debian/copyright to LICENSE.md and then regenerate debian/copyright when building for Debian (CMake infrastructure for that is mostly in place already, so this should be pretty easy to do).

Reply:
as discussed this morning during our developer meeting we will keep the symlink mechanism for the moment.

Reply:
> same as above, we could move debian/copyright to LICENSE.md and then regenerate debian/copyright when building for Debian (CMake infrastructure for that is mostly in place already, so this should be pretty easy to do).

Then we should have /LICENSE.md build like debian/copyright is created actually (so adding a tier lib etc will not got forgotten) and then copied around when debian packages are created.